### PR TITLE
FEATURE: Upgrade to fontawesome 5.12.1

### PR DESCRIPTION
**Pull request by markusguenther**
Fixes: #2880

### CODE REVIEW

**Comment by Sebobo** [comment]
@markusguenther  why WIP?

**Comment by albe** [comment]
I second that question :) Why WIP? What's required to get it finished? Can we get it into 5.2?

**Comment by markusguenther** [comment]
This PR was WIP because the Icons did not work in the backend. Figured now out why, fixed it and also updated to the latest version 5.12.1.

File: Neos.Neos/Resources/Public/Styles/Welcome.css [hunk 1]
```diff
@@ -25,3 +25,100 @@
 .neos-error-details-toggle--closed .neos-error-details-toggle__body {
   height: 0; }
 
+ule.exports, module, module.exports, __webpack_require__);
+/******/
```
Comment (daniellienert) [inline Neos.Neos/Resources/Public/Styles/Welcome.css]: Is this intentionally committed?

Comment (albe) [reply Neos.Neos/Resources/Public/Styles/Welcome.css]: Good catch!

File: Neos.Neos/Resources/Public/Styles/Welcome.css [hunk 2]
```diff
@@ -25,3 +25,100 @@
...
+/*! no static exports found */
+/*! all exports used */
+/*! ModuleConcatenation bailout: Module is not an ECMAScript module */
+/***/ (function(module, exports, __webpack_require__) {
+
+eval("// extracted by mini-css-extract-plugin//# sourceURL=[module]\n//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiOS5qcyIsInNvdXJjZXMiOlsid2VicGFjazovLy8uL1Jlc291cmNlcy9Qcml2YXRlL1N0eWxlcy9XZWxjb21lLnNjc3M/ZWI5MyJdLCJzb3VyY2VzQ29udGVudCI6WyIvLyBleHRyYWN0ZWQgYnkgbWluaS1jc3MtZXh0cmFjdC1wbHVnaW4iXSwibWFwcGluZ3MiOiJBQUFBIiwic291cmNlUm9vdCI6IiJ9\n//# sourceURL=webpack-internal:///9\n");
+
+/***/ })
+
+/******/ });
```
Comment (albe) [inline Neos.Neos/Resources/Public/Styles/Welcome.css]:
```suggestion
```

File: Neos.Neos/Resources/Public/Library/fontawesome/scss/_variables.scss [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
-$fa-font-path:                "../Library/fontawesome/webfonts" !default;
-$fa-font-size-base:           16px !default;
-$fa-font-display:             auto !default;
-$fa-css-prefix:               fa !default;
-$fa-version:                  "5.8.2" !default;
-$fa-border-color:             #eee !default;
-$fa-inverse:                  #fff !default;
-$fa-li-width:                 2em !default;
-$fa-fw-width:                 (20em / 16);
+$fa-font-path:         "../webfonts" !default;
```
Comment (albe) [inline Neos.Neos/Resources/Public/Library/fontawesome/scss/_variables.scss:4]: @markusguenther  is this path correct?

**Comment by albe** [comment]
Would merge tomorrow if no one objects.

**Comment by markusguenther** [comment]
Sry did not have so much time. Hope to look at some stuff at the weekend

**Comment by albe** [comment]
I'll merge this now - let's see what happens and fix any possible issues in the following two weeks :)

**Pull request merged**